### ISSUE TITLE
Add an option to calendar format to not include the time

### ISSUE THREAD

**Issue by kai670** [moment/moment#5215]
I need to show a date exactly like it is done by [calendar time](https://momentjs.com/docs/#/displaying/calendar-time/) but without the time. This seems to be a quite common problem as I've found at least two StackOverflow discussions but none of them provided a good solution.

For me the best solution would be adding a new `calendarDate` method or an option in current `calendar` method providing the same format but ignoring the time.

I tried:
 - Specifying calendar output format per invocation but it didn't work with localization.
 - Using `updateLocale` but that way I loose the possibility of using it to format with the time in other places and, additionally, it forces to replicate the locale information in the application.

**Comment by harkue** [comment]
I think there is a workaround solution which is to define all custom function to format the calendar like:
```js
moment().calendar(null, {
  sameDay: function (now) {
    return moment().format();
  }
});
```

**Comment by Alanscut** [comment]
@marwahaha @kai670 The feature has been implemented in `moment@2.25.0`. Please see https://momentjs.com/docs/#/displaying/calendar-time/.
